src/logoIcons/index.ts

```diff
@@ -0,0 +1,5 @@
+export { IconHorizontal } from './Horizontal';
+export { IconHorizontalInvert } from './HorizontalInvert';
+export { IconVertical } from './Vertical';
+export { IconVerticalInvert } from './VerticalInvert';
```

Comment:
These names are a bit unfortunate, they don't really describe that they are logo icons. But it's not a deal breaker

Reply:
Since some icons have been renamed already, these could probably be renamed as well. Good point 👍

Reply:
Added 'Logo' to the names, so they are now `IconLogoHorizontal` and so forth.